### PR TITLE
Status/2024Q1/bsdinstall.adoc: Add report

### PR DESCRIPTION
**Pull request by khorben**
Reflecting the current status of the integration, for a possible bootable image with a graphical installer to be generated from the base system.

Sponsored by:	The FreeBSD Foundation

### CODE REVIEW

File: website/content/en/status/report-2024-01-2024-03/bsdinstall.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Some parts are still missing, or made use of shortcuts still unsuitable for integration:
+
+* The "fetchmissingdists" target was avoided by shipping every component on the installation media;
+* The "checksum" and "extract" targets had to be re-implemented with simpler code, degrading the user experience also with the regular installer;
+* Creation of the installation media generates an additional, heavy image (almost 8 GB), and is suspected to be hindered by a bug in man:makefs[8].
+
+The corresponding code can be found in my link:https://github.com/khorben/freebsd-src/[GitHub fork] in the link:https://github.com/khorben/freebsd-src/tree/khorben/bsdinstall-graphical4[khorben/bsdinstall-graphical4] branch.
+As can be guessed from the branch name, depending on the complexity of rebasing operations, combined with the (hopefully) progressive integration of the changes proposed, new branches may be added to keep track of the progress.
+(In fact a link:https://github.com/khorben/freebsd-src/tree/khorben/bsdinstall-graphical5[khorben/bsdinstall-graphical5] branch already exists)
```
Comment (pauamma) [inline website/content/en/status/report-2024-01-2024-03/bsdinstall.adoc:54]: Needs "." before ")".

**Review by pauamma — APPROVED** [review]
Other than that, LGTM.

**Review by pauamma — APPROVED** [review]

**Review by lsalvadore — CHANGES_REQUESTED** [review]
Only two minor fixes and it is ready. If you prefer I can make the changes myself at merge time.

Thanks!

**Comment by lsalvadore** [comment]
Merged, thanks!

**Pull request closed without merging**